### PR TITLE
Updated PackagePublisher to publish the exe and related dlls

### PR DESCRIPTION
**Pull request by kichalla**
Keeping the things in sync with SplitPackages and making it work in the context of a Universe build.

### CODE REVIEW

File: src/PackagePublisher/PackagePublisher.nuspec [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>PackagePublisher</id>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <description>Publishes packages to a feed, with retries.</description>
+  </metadata>
+  <files>
+    <file src="net451\" target="/" />
```
Comment (pranavkm) [inline src/PackagePublisher/PackagePublisher.nuspec]: Why aren't we using a netcoreapp binary?

File: src/PackagePublisher/PackagePublisher.csproj [hunk 2]
```diff
@@ -1,13 +1,37 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\build\common.props" />
+  
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <Description>Publishes packages to a feed, with retries.</Description>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <TargetFrameworks>netcoreapp1.0;net451</TargetFrameworks>
```
Comment (pranavkm) [inline src/PackagePublisher/PackagePublisher.csproj]: Why do we need to cross-compile?

File: src/PackagePublisher/PackagePublisher.csproj [hunk 3]
```diff
@@ -1,13 +1,37 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\build\common.props" />
+  
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <Description>Publishes packages to a feed, with retries.</Description>
+    <VersionPrefix>1.0.0</VersionPrefix>
```
Comment (pranavkm) [inline src/PackagePublisher/PackagePublisher.csproj]: Can we use the same version as everything else in this repo (assuming they are all have the same version)?

File: src/PackagePublisher/PackagePublisher.nuspec (hunk 1 above)
Comment (kichalla) [inline src/PackagePublisher/PackagePublisher.nuspec]: I wondered the same but left it the same as SplitPackages. I will send out a separate PR for SplitPackage changes with netcoreapp1.0 only.

**Comment by kichalla** [comment]
🆙 📅 Could you also take a look at https://github.com/aspnet/Universe/pull/499

File: src/PackagePublisher/PackagePublisher.nuspec (hunk 1 above)
Comment (pranavkm) [inline src/PackagePublisher/PackagePublisher.nuspec]: Could you file a tracking item for this?

**Comment by pranavkm** [comment]
:shipit:

**Pull request merged**